psakefile.ps1

```diff
@@ -48,8 +48,10 @@ task VendorEditorServices {
   }
 
   Invoke-RestMethod -Uri $uri -OutFile $languageServerZip -ErrorAction Stop 
-  Expand-Archive -Path $languageServerZip -DestinationPath $langaugeServerPath -ErrorAction Stop 
+  Expand-Archive -Path $languageServerZip -DestinationPath "$($langaugeServerPath)/tmp" -ErrorAction Stop
+  Move-Item -Path (Join-Path $langaugeServerPath "tmp/$githubrepo-$githubref/*") -Destination $langaugeServerPath
```

Comment:
Typo   `langaugeServerPath` --> `languageServerPath`

Another 2 instances

Reply:
Whoops...that was MY original typo as well :-)

Reply:
fixed